src/function_lua.c

```diff
@@ -60,32 +62,56 @@ typedef struct luaFunctionCtx {
     int lua_function_ref;
 } luaFunctionCtx;
 
+typedef struct loadCtx {
+    libraryInfo *li;
+    monotime start_time;
+} loadCtx;
+
+static void luaEngineLoadHook(lua_State *lua, lua_Debug *ar) {
```

Comment:
let's add a comment saying what's it for and what it does.

Reply:
Fixed